zitadel/terraform/main.tf

```diff
@@ -0,0 +1,95 @@
+terraform {
```

Comment:
split into multiple files. 
provider.tf
and if it is possible to split each resource type to one file.. we must do that as well.

Reply:
okay.